CoreTableView/src/au/gov/asd/tac/constellation/views/tableview2/TableViewPane.java

```diff
@@ -827,17 +874,17 @@ private void loadPreferences() {
                 // Populate orderedColumns with full column ThreeTuples corresponding to entires in newVolumnOrder and call updateVisibleColumns
                 // to update table.
                 final List<ThreeTuple<String, Attribute, TableColumn<ObservableList<String>, String>>> orderedColumns
-                    = newColumnOrder.stream()
-                        .map(c ->
-                            {
-                                for (ThreeTuple<String, Attribute, TableColumn<ObservableList<String>, String>> col : columnIndex) {
-                                    if (c.getText().equals(col.getThird().getText())) {
-                                        return col;
+                        = newColumnOrder.stream()
+                                .map(c
+                                        -> {
```

Comment:
Move this arrow and open brace to the line above just for readability.
The `.map(c -> {` can then probably be moved to the line above that one (`... stream().map(...)`